addons/field_rations/initSettings.sqf

```diff
@@ -0,0 +1,44 @@
+[
+    QGVAR(enabled),
+    "CHECKBOX",
+    [localize ACELSTRING(common,Enabled), localize LSTRING(Enabled_Description)],
+    format ["ACEX %1", localize LSTRING(DisplayName)],
+    false,
+    true
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(timeWithoutWater),
+    "SLIDER",
+    [localize LSTRING(TimeWithoutWater_DisplayName), localize LSTRING(TimeWithoutWater_Description)],
+    format ["ACEX %1", localize LSTRING(DisplayName)],
+    [1, 24, 2, 0],
```

Comment:
For testing (or silly missions) I'd like to be able to set these lower (maybe down to 0.1 hour)